apps/provider-proxy/src/utils/telemetry.ts

```
@@ -0,0 +1,12 @@
+import type { Span } from "@opentelemetry/api";
+import { context } from "@opentelemetry/api";
+import { trace } from "@opentelemetry/api";
+
+export function traceActiveSpan(name: string, callback: (span: Span) => void): void {
+  return trace.getTracer("default").startActiveSpan(name, callback);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)